plugins/jira/help.md

```diff
@@ -14,6 +14,10 @@
 * Jira user email address and API key when using Jira Cloud
 * Jira username and password credentials when using on-prem Jira server
 
+# Supported Product Versions
+
+* 6.0
```

Comment:
```suggestion
* Jira Server 6.0
* Jira (Cloud)
* Jira ServiceDesk (Cloud)
```

Reply:
Just a heads up @mrinehart-r7 @mberezin-r7 - we should make sure we're thorough with versions going forward.

Reply:
@jschipp-r7 @r7-kszczepanskagorna make this change to the plugin.spec.yaml - not the help.md (otherwise this will get overwritten on regeneration)